cfme/tests/intelligence/reports/test_validate_chargeback_report_manual.py

```diff
@@ -0,0 +1,55 @@
+# pylint: skip-file
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream('upstream'), pytest.mark.manual]
```

Comment:
_Optional_ since all of these tests are chargeback tests you can put the `@test_requirements.chargeback` in this line

Reply:
Done.